schema.graphql

```diff
@@ -239,6 +239,7 @@ input FFIDMetadataInputMatches {
   extension: String
```

Comment:
After looking at the model again, the "file extension mismatch" param should be on the `FFIDMetadataInputValues` case class and should be remove, see line 35.